### DISCUSSION TITLE
Redash export to GSheet?

### DISCUSSION THREAD

**Discussion by michaelLovepop** [getredash/redash#6061] [Q&A]
Hello,

I'm wondering if redash can schedule exports a table to Google Sheets.  I believe it can but I can't find the documentation to confirm it.

Can some one here set me straight?

Thanks

**Comment by ayush-421** [comment]
Hey @michaelLovepop were you able to find the solution to this? I am also looking for the same thing. Please do share if you have found out how to do this.

Thanks!

**Comment by konnectr** [comment]
I think that now it is possible to import only here 
![image](https://github.com/getredash/redash/assets/46676677/99b9173b-0aa5-47f3-9f97-06128c9f7e9a)
